Rakefile

```diff
@@ -48,8 +167,69 @@ namespace :test do
     end
     Dir.chdir( original_dir )
   end
+
+  desc <<-EOS
+Run the base beaker acceptance tests
+#{USAGE}
+  EOS
+  task :base  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/base'})
+  end
+
+  desc <<-EOS
+Run the hypervisor beaker acceptance tests
+#{USAGE}
+  EOS
+  task :hypervisor  => 'gen_hosts' do
+    beaker_test({:tests => 'acceptance/tests/hypervisor'})
+  end
+
+  desc <<-EOS
+Run the puppet beaker acceptance tests on a pe install.
+#{USAGE}
+  EOS
+  task :puppetpe  => 'gen_hosts' do
+    beaker_test({:pre_suite => 'acceptance/pre_suite/pe/install.rb', :tests => 'acceptance/tests/puppet'})
```

Comment:
ah, yes.  i forgot about that issue when using merged_options.
i think in one of my project Rakefiles, when running in preserved_hosts mode i ensure we don't use the options file.  In that same project we use the options file(s) to set the presuite.
https://github.com/puppetlabs/orchestrator/blob/2016.2.x/acceptance/Rakefile#L71

will that fix the issue here?